setup.py

```diff
@@ -30,25 +31,28 @@ def local_open(fname):
 for directory in os.walk('example/templates'):
     example_dirs.append(directory[0][8:] + '/*.*')
 
+with local_open(os.path.join('pages', '__init__.py')) as f:
+    exec(f.read(), {}, pages)
```

Comment:
@greut I have tried with the latest fixes you have provided to `requirements-builder` and it works. I will release 0.1.1 ASAP. Thanks!